### DISCUSSION TITLE
Two NetFPGA Sume nics cannot ping each other.

### DISCUSSION THREAD

**Discussion by yanghjfis** [corundum/corundum#185] [Q&A]
hello,
I also used Corundum NIC on NetFPGA Sume board. Two ports were identified by ifconfig, but after SFP interface was connected directly, mutual ping failed. ethtool port, link detected was displayed: no, what should we do in this case, thank you very much

**Comment by alexforencich** [comment]
Did the design close timing (TNS of 0)?  What exactly are you using to connect the boards together?
